### ISSUE TITLE
[BUG] Bad behavior of overriddenDuration

### ISSUE THREAD

**Issue by 1x3x3x7** [jhomlala/betterplayer#538]
When you set `overriddenDuration: Duration(seconds: 30)` to `BetterPlayerDataSource` and this video is 10 seconds it have bad behavior. I think better solution is set max duration to progress bar and loop this video to the end.

**Comment by jhomlala** [comment]
Please read this: https://github.com/jhomlala/betterplayer/issues/535 and format your issue according to issue template.

**Comment by 1x3x3x7** [comment]
Sample of code:
```
class VideoScreen extends StatelessWidget {
  @override
  Widget build(BuildContext context) {
    return Scaffold(
      appBar: AppBar(
        title: Text("Video"),
      ),
      body: VideoWidget(),
    );
  }
}

class VideoWidget extends StatefulWidget {
  @override
  State<StatefulWidget> createState() {
    return _VideoWidgetState2();
  }
}

class _VideoWidgetState2 extends State<VideoWidget> {
  final playerConfig = BetterPlayerConfiguration(
      autoPlay: true,
      allowedScreenSleep: false,
      aspectRatio: 1280 / 720,
      controlsConfiguration: BetterPlayerControlsConfiguration(
          showControlsOnInitialize: false,
          enableOverflowMenu: false,
          enableSkips: false));

  @override
  Widget build(BuildContext context) {
    return Center(
        child: BetterPlayer(
            controller: BetterPlayerController(
      playerConfig,
      betterPlayerDataSource: createDataSource(),
    )));
  }

  BetterPlayerDataSource createDataSource() => BetterPlayerDataSource(
      BetterPlayerDataSourceType.network,
      'https://commondatastorage.googleapis.com/gtv-videos-bucket/sample/ForBiggerBlazes.mp4',
      overriddenDuration: Duration(seconds: 30));
}

**Comment by 1x3x3x7** [comment]
https://user-images.githubusercontent.com/85289655/121061677-d40fe780-c7cc-11eb-9965-afd8ce1601a4.mov

**Comment by jhomlala** [comment]
Fixed in 0.0.70.
